<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Jenny" />

<meta name="date" content="2019-12-20" />

<title>LD Score Regression</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/paper.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row-fluid">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">future_reference</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Topics
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="programming.html">Programming</a>
    </li>
    <li>
      <a href="softwares_data.html">Softwares and Data</a>
    </li>
    <li>
      <a href="learning_material.html">Learning Material</a>
    </li>
    <li>
      <a href="notes.html">Notes</a>
    </li>
  </ul>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/jennysjaarda/future_reference">
    <span class="fa fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">LD Score Regression</h1>
<h4 class="author">Jenny</h4>
<h4 class="date">2019-12-20</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2019-12-20
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>future_reference/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.5.0). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20191220code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20191220)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20191220code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20191220)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomjennysjaardafuturereferencetree43c40b69b93b7b3201002ffd47f0975d7933fd07targetblank43c40b6a"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> <a href="https://github.com/jennysjaarda/future_reference/tree/43c40b69b93b7b3201002ffd47f0975d7933fd07" target="_blank">43c40b6</a> </a>
</p>
</div>
<div id="strongRepositoryversionstrongahrefhttpsgithubcomjennysjaardafuturereferencetree43c40b69b93b7b3201002ffd47f0975d7933fd07targetblank43c40b6a" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility. The version displayed above was the version of the Git repository at the time these results were generated. <br><br> Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rproj.user/

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the R Markdown and HTML files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view them.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/jennysjaarda/future_reference/43c40b69b93b7b3201002ffd47f0975d7933fd07/docs/ldsr.html" target="_blank">43c40b6</a>
</td>
<td>
Jenny
</td>
<td>
2019-12-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/jennysjaarda/future_reference/e2df4055aaaf777d27fc4f5dcf64905e788f896a/docs/ldsr.html" target="_blank">e2df405</a>
</td>
<td>
Jenny
</td>
<td>
2019-12-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/jennysjaarda/future_reference/blob/89b229f4581ddc227b608465794e2aea8dc2fad6/analysis/ldsr.Rmd" target="_blank">89b229f</a>
</td>
<td>
Jenny
</td>
<td>
2019-12-20
</td>
<td>
Updating webiste
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/jennysjaarda/future_reference/bf770c287871430d8d5a989d58e1fb1c07607c37/docs/ldsr.html" target="_blank">bf770c2</a>
</td>
<td>
Jenny
</td>
<td>
2019-12-20
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/jennysjaarda/future_reference/blob/1d7bcab82e9ff2806909c2825e940369d0f4168f/analysis/ldsr.Rmd" target="_blank">1d7bcab</a>
</td>
<td>
Jenny
</td>
<td>
2019-12-20
</td>
<td>
Updating webiste
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="ld-score-regression-overview" class="section level1">
<h1>LD Score Regression Overview</h1>
<p>Raymond Walters, post-doc, BROAD</p>
<p><a href="https://www.youtube.com/watch?v=dVrF0l9jMgE">Part 1</a>, <a href="https://www.youtube.com/watch?v=QVPNouAbXsY">Part 2</a></p>
<p>Also useful: <a href="https://www.youtube.com/watch?v=rENZPgnppAg" class="uri">https://www.youtube.com/watch?v=rENZPgnppAg</a></p>
<p>LD Score regression lives in series of 3 papers in Nature Genetics (2015):</p>
<ol style="list-style-type: decimal">
<li><a href="https://www.nature.com/articles/ng.3404">Partitioning heritability by functional annotation using genome-wide association summary statistics</a></li>
<li><a href="https://www.nature.com/articles/ng.3211">LD Score regression distinguishes confounding from polygenicity in genome-wide association studies</a></li>
<li><a href="https://www.nature.com/articles/ng.3406">An atlas of genetic correlations across human diseases and traits</a></li>
</ol>
<div id="paper-1-univariate-heritability" class="section level2">
<h2>Paper 1: Univariate heritability</h2>
<div id="ld-score-motivation" class="section level3">
<h3>LD Score motivation</h3>
<ul>
<li>Stratification (i.e. false positives) and polygeneticity effects (i.e. real effects) look very similar.</li>
<li>How can we tell the difference? What portion of a life off the null line in a QQ plot is true vs. confounding?</li>
</ul>
</div>
<div id="some-intuition-on-ld-and-polygenicity" class="section level3">
<h3>Some intuition on LD and polygenicity</h3>
<ul>
<li>LD blocks exists of varying size.</li>
<li>Lonely SNPs (no LD) also exist.</li>
<li>In a GWAS, if an LD block harbors a causal variant, we will observe associations with each SNP in the LD block proportional to it’s r2 with the causal variant.</li>
<li>A SNP in an LD block is more likely to be associated with a trait proportional to the size of its respective LD block. .</li>
<li>Assuming a uniform prior, we see SNPs with more LD friends showing more associatin.</li>
<li>In other words, the more variation you tag (the more LD friends you have), the more likely you are to tag a causal variant simply because you have a higher chance of hitting a causal SNP.</li>
</ul>
<p><img src="assets/LD_score_LD_intuition.png" width="50%" style="display: block; margin: auto auto auto 0;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-LD_score_LD_intuition">
Past versions of LD_score_LD_intuition.png
</button>
</p>
<div id="fig-LD_score_LD_intuition" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/jennysjaarda/future_reference/blob/53fa8bd65b918e043571fd1c10ca65dc168ba1ce/docs/assets/LD_score_LD_intuition.png" target="_blank">53fa8bd</a>
</td>
<td>
Jenny
</td>
<td>
2019-12-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="some-intuition-on-ld-and-drift" class="section level3">
<h3>Some intuition on LD and drift</h3>
<ul>
<li>Assume primary concern for confounding is population stratification.
<ul>
<li>Over long periods of time you get a random walk of allele frequencies within each population.</li>
<li>Assuming recombination has not changed the underlying LD structure, then genetic drift will alter the AFs of each block together.<br />
</li>
<li>In other words, LD blocks will move together.</li>
<li>The liklihood that you have drifted is not related to the size of the block.</li>
<li>Under pure drift we expect LD to have no relatinship to difference in allele frequencies between populations.</li>
</ul></li>
</ul>
<p><img src="assets/LD_score_drift_intuition.png" width="50%" style="display: block; margin: auto auto auto 0;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-LD_score_drift_intuition">
Past versions of LD_score_drift_intuition.png
</button>
</p>
<div id="fig-LD_score_drift_intuition" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/jennysjaarda/future_reference/blob/53fa8bd65b918e043571fd1c10ca65dc168ba1ce/docs/assets/LD_score_drift_intuition.png" target="_blank">53fa8bd</a>
</td>
<td>
Jenny
</td>
<td>
2019-12-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Selection or differences in LD structure will cause the model to breakdown.</li>
</ul>
</div>
<div id="derivation-of-ld-score-regression" class="section level3">
<h3>Derivation of LD Score regression</h3>
<ul>
<li>Basic idea of an additive model: some collection of geneotypes form a phenotype (plus an error).</li>
<li>Standardize genotypes and assume beta’s are drawn from the same distribution across all j SNPs.</li>
<li>This implicitly assumes a relationship of beta with MAF (i.e. smaller MAFs equate to larger betas).</li>
<li>In a GWAS we estimate marginal effects (one SNP at a time) which can be broken down into three components: (1) bias from confounders, causal (or tagging of causal estimate), estimation error (captured by SE).</li>
<li>You get a sum over the effects of all the SNPs you are correlated with proportional to how correlated you are with that SNP</li>
<li><p>In a GWAS you get an underlying additional piece of information due to this correlation with other SNPs. <img src="assets/LD_score_derivation1.png" width="50%" style="display: block; margin: auto auto auto 0;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-LD_score_derivation1">
Past versions of LD_score_derivation1.png
</button>
</p>
<div id="fig-LD_score_derivation1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/jennysjaarda/future_reference/blob/53fa8bd65b918e043571fd1c10ca65dc168ba1ce/docs/assets/LD_score_derivation1.png" target="_blank">53fa8bd</a>
</td>
<td>
Jenny
</td>
<td>
2019-12-20
</td>
</tr>
</tbody>
</table>
</div>
</div></li>
<li>Assumptions of the model:
<ul>
<li>Expected beta is 0.</li>
<li>Variance of each beta is SNP heritability divided by total number of SNPs.<br />
</li>
<li>Beta’s are i.i.d.; i.e. SNPs themselves can be (and are) correlated, but the causal effect of SNP does not inform you of the causal effect of any other SNP (strong assumption).</li>
<li>Distribution of effect size’s are not related to LD (see intuition related to drift above, assumes no selection).</li>
<li>Effect size is not related to confounding.</li>
<li>Confounding is not related to error.</li>
</ul></li>
</ul>
<p><img src="assets/LD_score_assumptions.png" width="50%" style="display: block; margin: auto auto auto 0;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-LD_score_assumptions">
Past versions of LD_score_assumptions.png
</button>
</p>
<div id="fig-LD_score_assumptions" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/jennysjaarda/future_reference/blob/53fa8bd65b918e043571fd1c10ca65dc168ba1ce/docs/assets/LD_score_assumptions.png" target="_blank">53fa8bd</a>
</td>
<td>
Jenny
</td>
<td>
2019-12-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Convertion to chi-square and expectations:
<ul>
<li>Chi-square is equal to B^2/var(error).</li>
<li>var(e) is approximately 1/N.</li>
</ul></li>
</ul>
<p><img src="assets/LD_score_derivation2.png" width="50%" style="display: block; margin: auto auto auto 0;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-LD_score_derivation2">
Past versions of LD_score_derivation2.png
</button>
</p>
<div id="fig-LD_score_derivation2" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/jennysjaarda/future_reference/blob/53fa8bd65b918e043571fd1c10ca65dc168ba1ce/docs/assets/LD_score_derivation2.png" target="_blank">53fa8bd</a>
</td>
<td>
Jenny
</td>
<td>
2019-12-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Take expectation and sub in assumptions:
<ul>
<li>LD score: sum of your correlations with other SNPs.</li>
<li>Subbing into previous equation, we get the fundamental LD-score equation.</li>
</ul></li>
</ul>
<p><img src="assets/LD_score_derivation3.png" width="50%" style="display: block; margin: auto auto auto 0;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-LD_score_derivation3">
Past versions of LD_score_derivation3.png
</button>
</p>
<div id="fig-LD_score_derivation3" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/jennysjaarda/future_reference/blob/53fa8bd65b918e043571fd1c10ca65dc168ba1ce/docs/assets/LD_score_derivation3.png" target="_blank">53fa8bd</a>
</td>
<td>
Jenny
</td>
<td>
2019-12-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="estimation-with-ld-score-regression" class="section level4">
<h4>Estimation with LD score regression</h4>
<ul>
<li>We can regress chi-square statistics (from GWAS) on function of LD scores and sample for each SNP.</li>
<li>Intercept: as exstimate of 1 + Na
<ul>
<li>Test for deviation from 1 as index of stratification/confounding</li>
<li>Use to correct for confounding.</li>
</ul></li>
<li>Slope: estimate of SNP heritability divided by M.
<ul>
<li>With known M, can convert to estaimate of SNP heritability.</li>
<li>This is on observed scale (i.e. linear), need to make adjustment for binary traits.</li>
</ul></li>
<li>Standard errors are computed via block jackknife (since chi-squared are correlated due to LD). Jackknife is special case of the bootstrap.</li>
</ul>
</div>
<div id="deriving-the-ld-score" class="section level4">
<h4>Deriving the LD score</h4>
<ul>
<li>Use unbiased r2 estimator.</li>
<li>Referenec samples (usually 1000G Europeans).</li>
<li>LD score regression can be computed with out raw genetic data.</li>
<li>If raw data is available, can use GREML (variance component model based on genetic similiarity matrix in GCTA, akin to a twin study but in unrelated individuals) or Haseman-Elston instead.</li>
<li>Estimated SNP hertiability is variance explained by the <em>J</em> SNPs used to estimated LD score (i.e. SNPs in the reference panel), which can be different from the setof SNPs in the GWAS.</li>
<li>Assumption is that effect sizes in the GWAS extrapolate to reference SNPs (important if it’s a non-targeted chip, for example).</li>
<li>Best to restrict AF to MAF&gt;5% and exclude strong LD regions (e.g. MHC region).</li>
</ul>
</div>
</div>
</div>
<div id="paper-2-partitioning-heritability" class="section level2">
<h2>Paper 2: Partitioning heritability</h2>
<div id="motivation-for-partitioning" class="section level4">
<h4>Motivation for partitioning</h4>
<ul>
<li>We previously assumed the heritability for each SNP is on average the same genome-wide.</li>
<li>Many possible violations of this (coding vs. non-coding, type of gene/place of expression, AF).</li>
<li>Expland LD score regression model to evaluate whether there are regions f the genome with stronger effects.</li>
</ul>
</div>
<div id="a-model-for-partitioning-heritability" class="section level4">
<h4>A model for partitioning heritability:</h4>
<ul>
<li>Previous: var(Bj) = (SNP heritability)/M.</li>
<li>Generalize this to allow the variance to differ between categories.</li>
<li>With disjoint categories it’s the simplest case.</li>
<li>Otherwise we assume overlapping categories act additively on the total variance (e.g. in both a coding region and a conserved region).</li>
</ul>
<p><img src="assets/LD_score_partioning_h2_overview.png" width="50%" style="display: block; margin: auto auto auto 0;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-LD_score_partioning_h2_overview">
Past versions of LD_score_partioning_h2_overview.png
</button>
</p>
<div id="fig-LD_score_partioning_h2_overview" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/jennysjaarda/future_reference/blob/53fa8bd65b918e043571fd1c10ca65dc168ba1ce/docs/assets/LD_score_partioning_h2_overview.png" target="_blank">53fa8bd</a>
</td>
<td>
Jenny
</td>
<td>
2019-12-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Gives an LD score for each category.</li>
</ul>
<p><img src="assets/LD_score_partioning_h2_derivation.png" width="50%" style="display: block; margin: auto auto auto 0;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-LD_score_partioning_h2_derivation">
Past versions of LD_score_partioning_h2_derivation.png
</button>
</p>
<div id="fig-LD_score_partioning_h2_derivation" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/jennysjaarda/future_reference/blob/53fa8bd65b918e043571fd1c10ca65dc168ba1ce/docs/assets/LD_score_partioning_h2_derivation.png" target="_blank">53fa8bd</a>
</td>
<td>
Jenny
</td>
<td>
2019-12-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<ul>
<li>Estimate tau via multiple regression with LD score computed from reference.</li>
</ul>
<p><img src="assets/LD_score_overview.png" width="50%" style="display: block; margin: auto auto auto 0;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-LD_score_overview">
Past versions of LD_score_overview.png
</button>
</p>
<div id="fig-LD_score_overview" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/jennysjaarda/future_reference/blob/53fa8bd65b918e043571fd1c10ca65dc168ba1ce/docs/assets/LD_score_overview.png" target="_blank">53fa8bd</a>
</td>
<td>
Jenny
</td>
<td>
2019-12-20
</td>
</tr>
</tbody>
</table>
</div>
</div>
<p>See the following slides sets in “/slides” folder:</p>
<ol style="list-style-type: decimal">
<li><a href="https://www.med.unc.edu/pgc/statgen/stat-gen-archive-2015/">Brendan Bulik-Sullivan</a></li>
<li><a href="https://mesuturkey.wordpress.com/publications/">Mesut Erzurumluoğlu</a></li>
</ol>
<p>And this <a href="http://www.nealelab.is/blog/2017/9/14/heritability-501-ldsr-based-h2-in-ukbb-for-the-technically-minded">Neale blog</a></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 3.6.1 (2019-07-05)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS Catalina 10.15.2

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRblas.0.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] workflowr_1.5.0

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.3      rprojroot_1.3-2 digest_0.6.23   later_0.8.0    
 [5] R6_2.4.1        backports_1.1.5 git2r_0.26.1    magrittr_1.5   
 [9] evaluate_0.14   stringi_1.4.3   rlang_0.4.2     fs_1.3.1       
[13] promises_1.0.1  whisker_0.4     rmarkdown_2.0   tools_3.6.1    
[17] stringr_1.4.0   glue_1.3.1      httpuv_1.5.2    xfun_0.11      
[21] yaml_2.2.0      compiler_3.6.1  htmltools_0.4.0 knitr_1.26     </code></pre>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>


</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_').toLowerCase();
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
