---
title: "LD Score Regression"
author: "Jenny"
date: "2019-12-20"
output: workflowr::wflow_html
editor_options:
  chunk_output_type: console
---


# LD Score Regression Overview
Raymond Walters, post-doc, BROAD

[Part 1](https://www.youtube.com/watch?v=dVrF0l9jMgE), [Part 2](https://www.youtube.com/watch?v=QVPNouAbXsY)

Also useful: https://www.youtube.com/watch?v=rENZPgnppAg


LD Score regression lives in series of 3 papers in Nature Genetics (2015): 

1. [Partitioning heritability by functional annotation using genome-wide association summary statistics](https://www.nature.com/articles/ng.3404)
2. [LD Score regression distinguishes confounding from polygenicity in genome-wide association studies](https://www.nature.com/articles/ng.3211)
3. [An atlas of genetic correlations across human diseases and traits](https://www.nature.com/articles/ng.3406)

## Paper 1: Univariate heritability

### LD Score motivation
  - Stratification (i.e. false positives) and polygeneticity effects (i.e. real effects) look very similar.
  - How can we tell the difference? What portion of a life off the null line in a QQ plot is true vs. confounding?  
  
### Some intuition on LD and polygenicity 
  - LD blocks exists of varying size. 
  - Lonely SNPs (no LD) also exist. 
  - In a GWAS, if an LD block harbors a causal variant, we will observe associations with each SNP in the LD block proportional to it's r2 with the causal variant.
  - A SNP in an LD block is more likely to be associated with a trait proportional to the size of its respective LD block. .
  - Assuming a uniform prior, we see SNPs with more LD friends showing more associatin. 
  - In other words, the more variation you tag (the more LD friends you have), the more likely you are to tag a causal variant simply because you have a higher chance of hitting a causal SNP. 

```{r , echo=FALSE, out.width = '50%',fig.align='left'}
knitr::include_graphics("assets/LD_score_LD_intuition.png")
```


### Some intuition on LD and drift
  - Assume primary concern for confounding is population stratification.
  - Over long periods of time you get a random walk of allele frequencies within each population.
  - Assuming recombination has not changed the underlying LD structure, then genetic drift will alter the AFs of each block together.  
  - In other words, LD blocks will move together. 
  - The liklihood that you have drifted is not related to the size of the block.
  - Under pure drift we expect LD to have no relatinship to difference in allele frequencies between populations.

```{r echo=FALSE, out.width = '50%',fig.align='left'}
knitr::include_graphics("assets/LD_score_drift_intuition.png")
```

  - Selection or differences in LD structure will cause the model to breakdown. 
  
### Derivation of LD Score regression 
- Basic idea of an additive model: some collection of geneotypes form a phenotype (plus an error). 
- Standardize genotypes and assume beta's are drawn from the same distribution across all j SNPs. 
- This implicitly assumes a relationship of beta with MAF (i.e. smaller MAFs equate to larger betas).
- In a GWAS we estimate marginal effects (one SNP at a time) which can be broken down into three components: (1) bias from confounders, causal (or tagging of causal estimate), estimation error (captured by SE).
- You get a sum over the effects of all the SNPs you are correlated with proportional to how correlated you are with that SNP
- In a GWAS you get an underlying additional piece of information due to this correlation with other SNPs.
```{r , echo=FALSE, out.width = '50%',fig.align='left'}
knitr::include_graphics("assets/LD_score_derivation1.png")
```
   
- Assumptions of the model: 
  - Expected beta is 0.
  - Variance of each beta is SNP heritability divided by total number of SNPs.  
  - Beta's are i.i.d.; i.e. SNPs themselves can be (and are) correlated, but the causal effect of SNP does not inform you of the causal effect of any other SNP (strong assumption).
  - Distribution of effect size's are not related to LD (see intuition related to drift above, assumes no selection).
  - Effect size is not related to confounding.
  - Confounding is not related to error. 
  
```{r , echo=FALSE, out.width = '50%',fig.align='left'}
knitr::include_graphics("assets/LD_score_assumptions.png")
```

- Convertion to chi-square and expectations: 
  - Chi-square is equal to B^2/var(error).
  - var(e) is approximately 1/N.
  
```{r , echo=FALSE, out.width = '50%',fig.align='left'}
knitr::include_graphics("assets/LD_score_derivation2.png")
```
  
- Take expectation and sub in assumptions:
  - LD score: sum of your correlations with other SNPs. 
  - Subbing into previous equation, we get the fundamental LD-score equation. 

```{r , echo=FALSE, out.width = '50%',fig.align='left'}
knitr::include_graphics("assets/LD_score_derivation3.png")
```


#### Estimation with LD score regression
- We can regress chi-square statistics (from GWAS) on function of LD scores and sample for each SNP. 
- Intercept: as exstimate of 1 + Na
  - Test for deviation from 1 as index of stratification/confounding
  - Use to correct for confounding. 
- Slope: estimate of SNP heritability divided by M. 
  - With known M, can convert to estaimate of SNP heritability. 
  - This is on observed scale (i.e. linear), need to make adjustment for binary traits. 
- Standard errors are computed via block jackknife (since chi-squared are correlated due to LD). Jackknife is special case of the bootstrap.

#### Deriving the LD score
- Use unbiased r2 estimator.
- Referenec samples (usually 1000G Europeans).
- LD score regression can be computed with out raw genetic data.
- If raw data is available, can use GREML (variance component model based on genetic similiarity matrix in GCTA, akin to a twin study but in unrelated individuals) or Haseman-Elston instead.
- Estimated SNP hertiability is variance explained by the *J* SNPs used to estimated LD score (i.e. SNPs in the reference panel), which can be different from the setof SNPs in the GWAS. 
- Assumption is that effect sizes in the GWAS extrapolate to reference SNPs (important if it's a non-targeted chip, for example).
- Best to restrict AF to MAF>5% and exclude strong LD regions (e.g. MHC region).

## Paper 2: Partitioning heritability 

#### Motivation for partitioning 

- We previously assumed the heritability for each SNP is on average the same genome-wide. 
- Many possible violations of this (coding vs. non-coding, type of gene/place of expression, AF). 
- Expland LD score regression model to evaluate whether there are regions f the genome with stronger effects. 

#### A model for partitioning heritability: 

- Previous: var(Bj) = (SNP heritability)/M.
- Generalize this to allow the variance to differ between categories.
- With disjoint categories it's the simplest case. 
- Otherwise we assume overlapping categories act additively on the total variance (e.g. in both a coding region and a conserved region). 

```{r , echo=FALSE, out.width = '50%',fig.align='left'}
knitr::include_graphics("assets/LD_score_partioning_h2_overview.png")
```

- Gives an LD score for each category.

```{r , echo=FALSE, out.width = '50%',fig.align='left'}
knitr::include_graphics("assets/LD_score_partioning_h2_derivation.png")
```

- Estimate tau via multiple regression with LD score computed from reference.

```{r , echo=FALSE, out.width = '50%',fig.align='left'}
knitr::include_graphics("assets/LD_score_overview.png")
```

See the following slides sets in "/slides" folder:

  1. [Brendan Bulik-Sullivan](https://www.med.unc.edu/pgc/statgen/stat-gen-archive-2015/)
  2. [Mesut ErzurumluoÄŸlu](https://mesuturkey.wordpress.com/publications/)
  
And this [Neale blog](http://www.nealelab.is/blog/2017/9/14/heritability-501-ldsr-based-h2-in-ukbb-for-the-technically-minded)
  
  


